src/cli/repo/s3.go

```diff
@@ -128,6 +129,14 @@ func initS3Cmd(cmd *cobra.Command, args []string) error {
 		return Only(ctx, errors.Wrap(err, "Retrieving s3 configuration"))
 	}
 
+	if strings.Contains(s3Cfg.Endpoint, "http") {
```

Comment:
Maybe use `HasPrefix(..., "https?://")` instead of contains. `Contains` can be a bit fragile.